test/wait-for-it.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/env bash
```

Reply:
This caused:

`bredis_clusterer_1  | /usr/local/bin/docker-entrypoint.sh: 16: exec: /test/wait-for-it.sh: not found`

Reverted.